### PR TITLE
add LT

### PR DESCRIPTION
**Pull request by dw5**

### CODE REVIEW

File: languages/lt.json [hunk 1]
```diff
@@ -1,360 +1,515 @@
...
+      "pinned_by": "Prisegtas",
+      "upvoted_by": "Patinka",
+      "edited": "redaguotas",
+      "show_more_replies": "Rodyti daugiau atsakymų",
+      "show_all_replies": "Rodyti visus atsakymų",
+      "pin": "Prisegti"
+    },
+    "creator_portal": {
+      "creator_news": "Kūrėjų naujienos",
+      "creator_news_soon": "Creator News is coming soon!",
```
Comment (ByteOfMelon) [inline languages/lt.json]: I assume this was an accident? Your previous commit had translations for creator_news, creator_news_soon, etc

**Review by ByteOfMelon — CHANGES_REQUESTED** [review]
Unless it was intentional, the untranslated strings should be translated before this is merged.

**Comment by dw5** [comment]
fixed, plus some lines lacked `,` at the end.
did not have luck seeing lt in i18n-ally

**Comment by BlakeB415** [comment]
Thank you so much!

**Comment by ByteOfMelon** [comment]
Seems good to me now

**Pull request merged**